stream_alert/rule_processor/firehose.py

```diff
@@ -157,6 +157,23 @@ def _backoff_handler_firehose_reset(self, details):
         backoff_handler(debug_only=False)(details)
         self._firehose_client = self._create_firehose_client()
 
+    @staticmethod
+    def _strip_successful_records(batch, response):
+        """Inspect the response and remove any records records that have successfully to sent
```

Comment:
Can we add this to the docstring:

``` For each record, the index of the response element is the same as the index used in the request array.```

Reply:
From: http://boto3.readthedocs.io/en/latest/reference/services/firehose.html#Firehose.Client.put_record_batch